CONTRIBUTING.md

```diff
@@ -12,119 +10,81 @@ We'd love for you to contribute to our source code and to make Accord Project do
+* **Small Changes** can directly be crafted and submitted to the respective GitHub repository as a Pull Request. See the section about [Pull Request Submission Guidelines][contribute.submitpr], and for detailed information read the [core development documentation][developers].
 
 ### <a name="docs"></a> Want a Doc Fix?
 
-Should you have a suggestion for the documentation, you can open an issue and outline the problem
-or improvement you have - however, creating the doc fix yourself is much better!
-
-If you want to help improve the docs, it's a good idea to let others know what you're working on to
-minimize duplication of effort. Create a new issue (or comment on a related existing one) to let
-others know what you're working on.
-
-If you're making a small change (typo, phrasing) don't worry about filing an issue first. Use the
-friendly blue "Improve this doc" button at the top right of the doc page to fork the repository
-in-place and make a quick change on the fly. The commit message is preformatted to the right type
-and scope, so you only have to add the description.
-
-For large fixes, please build and test the documentation before submitting the PR to be sure you
-haven't accidentally introduced any layout or formatting issues. You should also make sure that your
-commit message follows the **[Commit Message Guidelines][developers.commits]**.
-
-### <a name="updating"></a> Updating Documentation through Docusaurus
-
-[Docusaurus][docusaurus] is used for maintaining our documentation. The process for this involves generation of
-markdown files concerning the API, as well as incrementing the docs version. These two steps are 
-done individually. Updates to the documentation are merged into master, which will generate the new
-markdown files. Once this is complete, a new pull request should be submitted with the version incremented.
-
-#### Step 1 - Generating Markdown
+Should you have a suggestion for the documentation, you can open an issue and outline the problem or improvement you have - however, creating the doc fix yourself is much better!
 
-Make updates to the source code to improve documentation. Merging this into master will automatically
-generate Markdown from JSDocs, most notably being the `cicero-api.md` and `ergo-api.md` files.
+If you want to help improve the docs, it's a good idea to let others know what you're working on to minimize duplication of effort. Create a new issue (or comment on a related existing one) to let others know what you're working on.
 
-#### Step 2 - Incrementing Version
+If you're making a small change (typo, phrasing) don't worry about filing an issue first. Use the friendly blue "Improve this doc" button at the top right of the doc page to fork the repository in-place and make a quick change on the fly. The commit message is preformatted to the right type and scope, so you only have to add the description.
```

Comment:
Does this exist/work?

Reply:
Ah I remember removing this from another repo a while ago, missed it here. It does not, so I'm removing it.